docker/Dockerfile-dev

```diff
@@ -12,7 +12,7 @@ ARG KUBECTL_VERSION="latest"
 ARG HELM_VERSION="latest"
 ARG MINIKUBE_VERSION="latest"
 ARG DAPR_CLI_VERSION="latest"
-ARG PROTOC_VERSION="21.1"
+ARG PROTOC_VERSION="21.12"
```

Comment:
Should this be auto-gen by the Makefile?

Reply:
I'll leave that for a separate PR :)